interfaces/builtin/basedeclaration.go

```diff
@@ -409,6 +409,11 @@ slots:
     allow-installation:
       slot-snap-type:
         - core
+  password:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
The base declaration LGTM. The interface is privileged, but it does exactly what it says it does.